<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1366</width>
    <height>768</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1366</width>
    <height>768</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color : white</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color : white;</string>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>1366</width>
      <height>768</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page_1">
     <widget class="QLabel" name="lblClock">
      <property name="geometry">
       <rect>
        <x>1070</x>
        <y>670</y>
        <width>291</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Horloge : 50 s</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="lblNumCompleted">
      <property name="geometry">
       <rect>
        <x>1060</x>
        <y>600</y>
        <width>291</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QWidget" name="running_process" native="true">
      <property name="geometry">
       <rect>
        <x>1030</x>
        <y>210</y>
        <width>311</width>
        <height>261</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color : white;
border : 2px solid red;
border-radius : 10px
</string>
      </property>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>20</y>
         <width>141</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border : 0px</string>
       </property>
       <property name="text">
        <string>En traitement</string>
       </property>
      </widget>
      <widget class="QLabel" name="rp_id">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>70</y>
         <width>211</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border : 0px</string>
       </property>
       <property name="text">
        <string>ID : </string>
       </property>
      </widget>
      <widget class="QLabel" name="rp_burst_left">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>170</y>
         <width>241</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border : 0px</string>
       </property>
       <property name="text">
        <string>Temps restant : </string>
       </property>
      </widget>
      <widget class="QLabel" name="rp_priority">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>120</y>
         <width>251</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border : 0px</string>
       </property>
       <property name="text">
        <string>Priorité</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="rp_waiting_time">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>220</y>
         <width>241</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border : 0px</string>
       </property>
       <property name="text">
        <string>Temps attendu : </string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="algoName">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>10</y>
        <width>500</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>17</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>660</y>
        <width>811</width>
        <height>81</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,0">
       <property name="spacing">
        <number>30</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="btnPlay">
         <property name="font">
          <font>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Demarrer</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnPause">
         <property name="font">
          <font>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Pause</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnStop">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Arrêter</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="waitingWidget" native="true">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>981</width>
        <height>531</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget#waitingWidget {
    border: 2px solid blue;
    border-radius: 10px;
    background-color:  rgb(250 , 250, 250 );
}

/* Réinitialiser la bordure pour tous les enfants directs */
QWidget#waitingWidget &gt; * {
    border: none;
	 background-color:  rgb(248 , 248, 248 );
}</string>
      </property>
      <widget class="QWidget" name="gridLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>961</width>
         <height>511</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gl1"/>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="lblStatsTitle">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>20</y>
        <width>481</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border  : none; 
</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QTextEdit" name="txtStatsTable">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>871</width>
        <height>281</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border :none;</string>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QWidget" name="widgetChartWait" native="true">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>400</y>
        <width>421</width>
        <height>301</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="widgetChartDeadlines" native="true">
      <property name="geometry">
       <rect>
        <x>910</x>
        <y>100</y>
        <width>431</width>
        <height>281</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lblRecommandations">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>440</y>
        <width>781</width>
        <height>261</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border : 1px solid green ;
border-radius : 5px;</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="textFormat">
       <enum>Qt::RichText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1366</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
